count_files/utils/help_text.py

```diff
@@ -109,23 +95,138 @@
     't', 'total',
 ]
 
-group_names = [
-    # certain group description
-    'cg', 'count-group', 'sg', 'search-group', 'tg', 'total-group',
-    # all group descriptions
-    'groups'
-]
 search_words = [
     # sorting by purpose
     'service', 'common', 'special',
     # sorting by argument type
-    'positional', 'optional',
+    'positional', 'optional'
+]
+
+group_names = [
+    # certain group description
+    'cg', 'count-group', 'count',
+    'sg', 'search-group', 'search',
+    'tg', 'total-group', 'total',
+    # all group descriptions
+    'group'
     # sorting by group, including group description
-    'count', 'search', 'total',
 ]
 
+docs_text = f"""HELP SYSTEM EXTENSION DOCS.
+
+BASIC USAGE EXAMPLES:
+Topic - argument or group name, certain words for searching or sorting.
+    count-files -ah <topic>
+    count-files --args-help <topic>
+Show short help text: Topic must be in upper case or with one letter in upper case.
+    count-files --args-help <topic in upper case>
+Show more detailed help text: Topic must be in lower case.
+    count-files --args-help <topic in lower case>
+Search by short/long argument name:
+    count-files --args-help st
+    count-files --args-help supported-types
+Sorting arguments by group, including group description:
+(count, search or total)
+    count-files --args-help count
+ADDITIONAL SECTIONS:
```

Comment:
There should be a blank line before these CAPs style headings.

Reply:
No blank lines to take up less space. You can add where it is needed. Displaying the list of topics as a string also for using less space.